docs/windows.md

```diff
@@ -201,6 +201,15 @@ Download and apply `antrea-windows.yml`.
 kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-windows.yml
 ```
 
+Since Antrea 1.10, in kube-proxy mode, you can also deploy antrea-agent Windows DaemonSet with containerd runtime by applying file `antrea-windows-containerd.yml`.
+
+Download and apply `antrea-windows-containerd.yml`.
+
+```bash
+# Example:
+kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-windows-containerd.yml applying file `antrea-windows-containerd.yml`.
```

Comment:
Do you need to update the assets if you expect users to download the antrea-windows-containerd.yaml in a release?

Reply:
Updated, thanks.